packages/dds/test-dds-utils/src/ddsFuzzHarness.ts

```diff
@@ -651,7 +669,9 @@ export function mixinAttach<
 						state.containerRuntimeFactory,
 						clientA,
 						model.factory,
-						index === 0 ? "summarizer" : makeFriendlyClientId(state.random, index),
+						index === 0 && clientA.channel.id !== "summarizer"
```

Comment:
i'm a little confused about why anything needs to change in the attach handling. maybe it was debug code that can be reverted?

Reply:
I think this is where some of the client list handling needs to be done - previously, when a detached client was created and stored in the state.clients, it had a letter id and was not the summarizer client (which lead to the assignment to summarizerClient and swap of the first element in clients a few lines below this). Now, the detached client is the summarizer client, which attach didn't expect before, so I updated the expectations. This might not be the correct approach, but that was my understanding of what happened previously.